spot-client/src/common/backend/utils.js

```diff
@@ -229,6 +229,44 @@ export function getRemotePairingCode(serviceEndpointUrl, jwt) {
     });
 }
 
+/**
+ * Checks if it's ok to call given phone number.
+ *
+ * @param {string} serviceEndpointUrl - The URL pointing to the phone authorize service.
+ * @param {string} phoneNumber - E.164 formatted phone number to be checked.
+ * @returns {Promise<{allow: boolean, message: string}>}
+ */
+export function phoneAuthorize(serviceEndpointUrl, phoneNumber) {
+    if (!serviceEndpointUrl) {
+        throw new Error('phoneAuthorize - serviceEndpointUrl is required');
```

Comment:
The function above returns Promise.reject and the function below throws errors. Which one would you prefer? Is there a difference?

Reply:
yeah, it needs to be rejected promise